.travis.yml

```diff
@@ -6,5 +6,9 @@ python:
   - "3.3"
   - "3.4"
   - "pypy"
+sudo: false
 install: pip install -r requirements.txt
-script: nosetests
+script:
+  - py.test -v --cov flake8diff --cov-report term-missing
```

Comment:
@gregarmer maybe do flake8 check here as well. after all this is flake8 repo so should probably adhere to flake8 as well...